lib/src/components/NxIconDropdown/__tests__/NxIconDropdown.test.tsx

```diff
@@ -92,20 +93,21 @@ describe('NxIconDropdown', () => {
       <button id="link3" className="nx-dropdown-button" key="3">Link3</button>,
       <button id="link4" className="nx-dropdown-right-button" key="4">Link4</button>
     ];
-    const component = getShallowComponent({ children, isOpen: true }),
-        menu = component.find(NxDropdownMenu);
+    const component = getMountedComponent({ children, isOpen: true }),
+        menu = component.find(NxDropdownMenu).find('div').at(0);
```

Comment:
Similar to a suggested change I made on `NxDropdown`, I'd use a single selector string here using both the tag name and class name.

Reply:
> Similar to a suggested change I made on `NxDropdown`, I'd use a single selector string here using both the tag name and class name.

Updated it here: 49aa3582e3f83104b1fe96ee9d123b417d169b53